frontend/app/routes/login.tsx

```diff
@@ -27,8 +27,13 @@ export async function action({ request }: ActionFunctionArgs) {
   let session = await sessionStorage.getSession(request.headers.get("cookie"));
   session.set("user", user);
 
+  let backendCookie = ""
+
+  if (user.backendCookie) 
+    backendCookie = ", " + user.backendCookie;
+
```

Comment:
```suggestion
let backendCookie: string[] = [];

  if (user.backendCookie)
    backendCookie = user.backendCookie;

  let headers = new Headers({ "set-cookie": await sessionStorage.commitSession(session) });

  backendCookie.forEach((cookie: string) => {
    headers.append("set-cookie", cookie);
  });
```
treat backendCookie as array and add separate set-cookie header for each cookie